### ISSUE TITLE
Allow to turn off online mode via the config

### ISSUE THREAD

**Issue by Geolykt** [tazz4843/McPy#74]
See above, it was frequently requested, so it only makes sense to add it (and helps me to debug things when I have no internet (which is almost always))

**Comment by hydrostaticcog** [comment]
Agreed. Quarry supports offline mode, but it was never implemented in our network code. I am working on implemented as we did for the max players and motd.
